### PR TITLE
Feature/remove module titles

### LINKED ISSUE #3131: Remove Modules titles
**Describe the solution you'd like**
As UX improvement we will remove the title of each module

**Current**
image

**Required**
image

### PR DESCRIPTION
**Pull request by pablomarga**
Hi team this PR remove the title from all the modules.

Captura de pantalla 2021-04-14 a las 13 56 10

Closes #3131

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the Wazuh app project will be documented in this file.
 
+## Wazuh v4.2.0 - Kibana 7.10.0 , 7.10.2 - Revision 4202
+
+### Added
+- Remove module titles [#3160](https://github.com/wazuh/wazuh-kibana-app/pull/3160)
```
Comment (Desvelao) [inline CHANGELOG.md:8]: Maybe, It should be in `Changed` instead of `Added` changelog section

**Comment by Desvelao** [comment]
Test
I found a possible problem:
image
How to reproduce:
- Go to some Module.
- Click the info button near to module name in the breadcrumb.
- Scroll using the mouse wheel

I am not sure if this could be fixed or maybe we can replace it by a `EuiTooltip` instead of `EuiPopover`?

The next thing could not be considered a problem, but the module name is rendered first as test and after appear the Module info popover

**Comment by eze9252** [comment]
Is this ok?

Captura de pantalla de 2021-04-27 12-41-23

How to reproduce:

- Select an agent
- Go to somo module
- Check the breadcrumb

**Review by CPAlejandro — APPROVED** [review]
LGTM!

**Review by frankeros — APPROVED** [review]
LGTM!

**Pull request merged**